### ISSUE TITLE
Profile: Sprint 9 (4.26.23 - 5.9.23)

### ISSUE THREAD

**Issue by BerniXiongA6** [department-of-veterans-affairs/va.gov-team#56506]
## Sprint 8 Retro
EasyRetro Board ([link](https://easyretro.io/board/f4051524-d7be-43a4-8c50-f43d887f3a7f/d62963e1-2e23-488c-9abc-3cb50a92ad05))
## Sprint 9 Notes
Sprint 9 epic was updated using this [project outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/authenticated-experience/sprint-objectives/profile/sprints-1-10-Jan23-May23.md#sprint-9-426---59) 

## 2023 Priorities for Authenticated Experience
High: Migrating MyHealtheVet into VA.gov; expanding notification functionality (both onsite and preferences); improving the logged-in nav/IA/experience overall; urgent/required maintenance for a feature; unplanned work that aligns with OCTO high-priority objectives
Low: improvements team wants to make but might fall outside of OCTO objectives
To complete: some initiatives that we should finish but don't fall under OCTO high-priority objectives

## Q2 CY2023 (Apr-Jun) Product Goals
More details found at [Q2 CY2023 Product Roadmap](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/authenticated-experience/roadmap/2023-goals-roadmap.md#q2-cy2023-april---june)

## PTO
- Liz: 5/3-5/5
- Adam: 5/4 - 5/9 (sister is getting married 🥳 )

## Sprint 9 Goals
_At the end of this sprint, what do we want to have accomplished/delivered?_
- **Direct deposit migration EVSS > Lighthouse:** FE integration and pre-QA testing completed before whole team runs QA tasks in the following sprint.
- **Notification preferences:** PM performs action items from Recommended Touch Points / Collab Cycle requirements; finish updating notif. pref. feature flag filters on FE; BE communication preferences - document default opt-in flag.
- **Logged-in IA/nav research:** Finish consolidated report on all research and next steps w/ Mikki and Chante (will hold on further work beyond this until Samara returns).
- **Unplanned work** If needed, allot 2-3 pts for this sprint.
- **General/Other** Design, if time, can update documentation on mobile number flow (research recommendations). FE deprecate React components; BE remove email change toggle. Travis continue onboarding activities and taking on Product tasks. 

##  Direct deposit migration EVSS > Lighthouse (Epic #51900)

**Move QA tasks to Sprint 10**

_Carry-over:_
_BE
- [x] Validate - Config and test metrics and error logging (1 pt left) #56706

_BE/FE_
- [ ] In Progress - Figure out how to locally test outlier use cases (1 pt left) #54879

BE / FE
**If not ready to run QA tasks, what dev work or integration testing is left?**
- [x] Finish FE initial adapter rollout work (1 pt left) - https://github.com/department-of-veterans-affairs/va.gov-team/issues/56569
- [ ] FE lighthouse api revision (3 pts) - https://github.com/department-of-veterans-affairs/va.gov-team/issues/57407
- [ ] BE (4 pts) - #57359
- [ ] BE (2 pts) -  #57357
- [ ] BE (2 pts) -  #57361
- [x] BE (2 pts) - #57419

## Notification preferences
PM (w/ help of team's input)
- [ ] Perform action items from Collab Cycle KO ticket #57409 (refer to Recommended Touch Points section of the ticket)
- [ ] #56397 

_Carry-over:
_FE_
- [ ] Currently Blocked - update notif. pref. so that feature flag can filter by channel (email/text) (1 pt left) #55657

BE
Default send technical discovery questions: **(BX: See paragraph below for Tom's answers)**
- What is required to consume default opt-in indicator?
- While the UI may not change, does this require any FE changes in terms of how FE connects to BE, or is this entirely BE work? 
- Confirm whether default send work needs to happen before Adam can start on FE updates based on Liz's designs.

_There doesn't seem to be any BE work required - the default opt-in indicator is already present in the responses from vets-api.   Here are the [discovery notes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/notification-preferences/discovery-and-research/default-send/july-2022-notes.md).  But the BE documentation does not include this indicator._ - Tom

BE
- [x] (1 pt) - #57421

## Logged-in IA/nav research (Epic #51519)

Design
Carry-over:
- [x] Validate (waiting for feedback from Product Lead) - Complete research synthesis (3 pts left) #56722
- [x] Finish consolidated report on all research efforts - 8 pts - #57090
- [x] Talk about next steps with Mikki and Chante (this could be anything from more work in this area to putting this on hold until Samara is back from leave) - 3 pts - #57091

## General / Other
Design
- [ ] #48952 - Research Recommendations | Update Documentation on Mobile Number Flow (3 pts) - (should be unblocked by Sprint 9 but can be pushed out to Sprint 10 if needed)

FE
- [x] https://github.com/department-of-veterans-affairs/va.gov-team/issues/57344 - Deprecating React components / migrate to Web Comps (2pts)

BE
- [ ] Remove Email change toggle #54082 (2 pts)

## Unplanned Work
- For this sprint -- allot 2-3 points for unplanned work
